.github/workflows/integration.yaml

```diff
@@ -40,4 +40,5 @@ jobs:
         working-directory: .tests/integration
         run: |
           mamba install -c conda-forge -c bioconda singularity=3.8.6
-          snakemake -s ../../workflow/Snakefile -j 1 --show-failed-logs  --configfile config.yaml --use-singularity --singularity-args  " --cleanenv --bind /home/runner "
+          snakemake -s ../../workflow/Snakefile -j 1 --show-failed-logs --configfile config.yaml --use-singularity --singularity-args "--cleanenv --bind /home/runner"
+          snakemake -s ../../workflow/Snakefile -j 2 --show-failed-logs --configfiles config_pacbio.yaml --use-singularity --singularity-args "-e --cleanenv"
```

Comment:
This integration test is failing due to missing input files for paraphase.

Reply:
They have been added, but paraphase will not work as intended before the PR https://github.com/snakemake/snakemake-wrappers/pull/3071 is accepted